docs/development/release.rst

```diff
@@ -6,74 +6,58 @@
 How to make a Gammapy release
 *****************************
 
-This page contains step-by-step instructions how to make a Gammapy release.
+This page contains step-by-step instructions how to make a Gammapy release. We mostly follow the
+`Astropy release instructions <https://docs.astropy.org/en/latest/development/releasing.html>`__
+and just lists the additional required steps.
 
-Overview
---------
 
-We have structured the procedure in three phases:
+Feature Freeze and Branching
+----------------------------
 
-#. "Pre release" -- on a day (or several days) before making the release
-#. "Make release" -- on the day of making the release
-   (tag the stable version and make source release on PyPI)
-#. "Post release" -- on the day when announcing the release
-   (ideally only two or three days after making the release)
+#. Follow the `Astropy feature freeze and branching instructions <https://docs.astropy.org/en/latest/development/releasing.html#start-of-a-new-release-cycle-feature-freeze-and-branching>`__
+   Instead of updating the `whatsnew/<version>.rst` update the `docs/release-notes/<version>.rst`.
 
-The purpose of writing the procedure down explicitly is to make it easy for
-anyone to make a release (as opposed to one or a few people with secret
-knowledge how to do it). It's also good to not have to remember this stuff, and
-to avoid errors or forgotten steps.
 
-Note that we currently don't do bugfix releases, i.e. making a release is always simple in the sense
-that you just need to create a branch off of ``master``.
-Making bugfix releases would be more difficult and involve identifying commits with bug fixes and
-backporting those to the stable branches.
+Releasing the first major release candidate
+-------------------------------------------
 
-In these notes we'll use the Gammapy 0.14 release as an example.
+#. Add an entry for the release candidate like `v1.0rc1` or `v1.1rc1` in the ``download/index.json`` file in the `gammapy-web repo <https://github.com/gammapy/gammapy-webpage>`__.
+   The ``datasets`` entry should point to dev version of `gammapy-data`. This allows to fix bugs in the data during the release candidate testing.
```

Comment:
mention that notebooks can be left to the same value as dev?

Reply:
Done